docs/catalogue/cat_cohort-data-manager.md

```diff
@@ -213,39 +225,45 @@ is in a cohort or in a data bank.
 
 | *Column name* | *Description* | *Remarks* |
 | --- | --- | --- |
+| fromDataDictionary.resource \* | Source cohort pid. |  |
+| fromDataDictionary.version \* | Source data dictionary version. | |
 | fromTable \* | Source table name. | Tables must be predefined in the *SourceTables* sheet. |
-| toNetwork \* | Name of the target network or common data model. | e.g. LifeCycle, LongITools or ATHLETE or ConcePTION_WP7 |
+| toDataDictionary.resource \* | Name of the target common data model.  | e.g. LifeCycle_CDM, LongITools_CDM |
+| toDataDictionary.version \* | Source data dictionary version. | |
 | toTable \* | Target table name. | Map to a table that is defined in a common data model. |
 | description | Description of the harmonisation. | |
 | syntax | Syntax used for this harmonisation. | |
 
-<sup>*Table 7. Description of the columns that can be filled out for VariableMappings. \* = mandatory</sup>
-
-![Cohort table mapping](../img/cat_cohort-table-mappings-sheet.png)
-
-<sup>*Figure 9. Example of TableMappings filled out in Excel.*</sup>
+<sup>*Table 8. Description of the columns that can be filled out for VariableMappings. \* = mandatory</sup>
 
 #### *VariableMappings* sheet
 
 Harmonisation procedures at the variable level are defined in the *VariableMappings* sheet.
 
 | *Column name* | *Description* | *Remarks* |
 | --- | --- | --- |
+| fromDataDictionary.resource \* | Source cohort pid. |  |
+| fromDataDictionary.version \* | Source data dictionary version. | |
 | fromTable \* | Source table name. | Tables must be predefined in the *SourceTables* sheet. |
 | fromVariable | Source variable name(s). | Variables must be predefined in the _
```

Comment:
markdown table breaks after 'in the _'
also column: fromVariablesOtherTables.table, fromVariablesOtherTables.name